examples/ecs_with_ec2/backend.tf

```diff
@@ -1,2 +0,0 @@
-terraform {
```

Comment:
You have removed almost all files from the ecs_with_ec2 example. Please re-add. Otherwise, the example does not work.

Reply:
Done